eventstore/datadog_checks/eventstore/eventstore.py

```diff
@@ -36,16 +35,15 @@ def check(self, instance):
             raise CheckException('Invalid Status Code, {0} returned a status of {1}.'.format(url, r.status_code))
         # Unable to deserialize the returned data
         try:
-            parsed_api = json.loads(r.text)
+            parsed_api = r.json()
```

Comment:
does this raise the same error? I think so but it's good to confirm

Reply:
Yes

Reply:
actually no, py3 json has its own error `JSONDecodeError`

Reply:
ahh great!